src/asciidoc/docs/customizations/media.adoc

```diff
@@ -0,0 +1,220 @@
+= Media
+:page-slug: media
+:page-description: Standard VCC for selecting an image, file, sound, or video.
+:figure-caption!:
+
+Users can
+//tag::description[]
+select an image, file, sound, or video by selecting from the available options, uploading, or entering a URL.
+//end::description[]
+
+image::mediaVCC.png[title="Media VCC"]
+
+== Sample media VCC
+
+[source,json]
+----
+{
+    "key": "imageOrVideo",
+    "name": "Image or Video",
+    "type": "media",
+    "typeOptions": {
+        "acceptOnly": ["image", "video"],
+        "videoOptions": {
+            "hls": true
+        }
+    }
+}
+----
+
+== Type options
+
+The following code sample shows all the available `typeOptions`.
+Each option is described in the sections that follow the sample.
+
+[source,json]
+----
+{
+    "name": "Media Upload",
+    "type": "media",
+    "typeOptions": {
+        "acceptOnly": ["image", "video", "audio"],
+        "returnType": "url"|"extended",
+        "hideExtensions": false,
+        "videoOptions": {
+            "hls": true,
+            "estimatePoses": false
+        },
+        "imageOptions": {
+            "width": 100,
+        },
+        "audioOptions": {},
+        "fileOptions": {}
+    }
+}
+----
+
+=== acceptOnly
+
+`acceptOnly` allows you to specify the VCC types that are allowed.
+The value for `acceptOnly` is an array of one or more VCC types.
+If the array contains more than one type, then the user can choose any file of a type that matches one of the types in the array.
+For example, `"acceptOnly": ["image", "video"]` allows the user to select an image file or a video file.
+If the value of `acceptOnly` is an empty array, or if `acceptOnly` is not included in `typeOptions`, then any file type will be allowed.
+
+The possible types are as follows.
+
+* *image* - If you set `acceptOnly` to an array containing `image`, the user will be prompted to choose an image file for the VCC.
+Using a `media` VCC with `acceptOnly` set to `["image"]` is equivalent to using an <<image#, image VCC>>.
+
+* *video* - If you set `acceptOnly` to an array containing `video`, the user will be prompted to choose a video file for the VCC.
+Using a `media` VCC with `acceptOnly` set to `["video"]` is equivalent to using a <<video#, video VCC>>.
+
+* *audio* - If you set `acceptOnly` to an array containing `audio`, the user will be prompted to choose a sound file for the VCC.
+Using a `media` VCC with `acceptOnly` set to `["audio"]` is equivalent to using a <<sound#, sound VCC>>.
+
+* *file* - If you set `acceptOnly` to an array containing `file`, the user will be prompted to choose a file for  the VCC.
+The file selected can be of any type.
+Using a `media` VCC with `acceptOnly` set to `["file"]` is equivalent to using a <<file#, file VCC>>.
+
+=== hideExtensions
+
+The `media` VCC type enables you to hide all asset packs and VCC extensions in cases where they don't make sense (for example, templates for selling premium assets).
```

Comment:
Question about how this works vs. using hideExtensions on the specific types. E.g., can you use imageOptions.hideExtensions to hide image VCCs but keep sound VCCs? I expect this would work, but I'm not sure what the use case would be. Maybe we'll just want to clarify that the top-level option applies to extensions for all allowed types.